pkg/api/scylla/v1/types_cluster.go

```diff
@@ -323,6 +323,7 @@ type SchedulerTaskSpec struct {
 	StartDate string `json:"startDate,omitempty"`
 
 	// interval represents a task schedule interval e.g. 3d2h10m, valid units are d, h, m, s.
+	// Deprecated: please use cron instead.
```

Comment:
this is not in scope in the description of this PR, please split this so its visible in the release notes

Reply:
ok, https://github.com/scylladb/scylla-operator/pull/1853